sktime/forecasting/arch/_uarch.py

```diff
@@ -398,19 +398,19 @@ def _predict_interval(self, fh, X, coverage):
             upper_int = mean_forecast + (z_critical * std_err)
             lower_df = pd.DataFrame(
                 lower_int,
-                columns=[y_col_name + " " + str(alpha) + " " + "lower"],
+                columns=[str(y_col_name) + " " + str(alpha) + " " + "lower"],
```

Comment:
we should not convert to str - convention is, names should be identical to column names in the input `y`

Reply:
Kindly review the changes.